enterprise/api/v1/views.py

```diff
@@ -171,8 +172,13 @@ def courses(self, request, pk=None):  # pylint: disable=invalid-name,unused-argu
         serializer.update_enterprise_courses(enterprise_customer, catalog_id=enterprise_customer.catalog)
         return get_paginated_response(serializer.data, request)
 
-    @detail_route(methods=['post'])
-    def course_enrollments(self, request, pk):  # pylint: disable=invalid-name,unused-argument
+    @detail_route(methods=['post'], permission_classes=[
+        permissions.IsAuthenticated,
+        permissions.DjangoModelPermissions,
```

Comment:
we need to remove this particular permission check for this endpoint

Reply:
I removed but a certain (pre-existing) test was failing on that. So I added that to that end point.

Reply:
We should adjust the test then. This new permission class is meant to replace this one for this endpoint.